converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/Worker/ChangeFeedProcessor.cs

```diff
@@ -53,9 +58,18 @@ public async Task ProcessAsync(TimeSpan pollIntervalDuringCatchup, CancellationT
             while (true)
             {
                 // Retrieve the change feed for any changes.
-                IReadOnlyList<ChangeFeedEntry> changeFeedEntries = await _changeFeedRetrieveService.RetrieveChangeFeedAsync(
-                    state.SyncedSequence,
-                    cancellationToken);
+                IReadOnlyList<ChangeFeedEntry> changeFeedEntries;
+                try
+                {
+                    changeFeedEntries = await _changeFeedRetrieveService.RetrieveChangeFeedAsync(
+                        state.SyncedSequence,
+                        cancellationToken);
+                }
+                catch (Exception ex)
```

Comment:
Generally we should not be catching all exceptions like this. What is the pattern followed in dicom-cast to catch these exceptions and logging it? We use middleware to handle and log in dicom-server

Reply:
I  didn't see something like middleware in dicom cast.
Exception is log and thrown like this: https://github.com/microsoft/dicom-server/blob/master/converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/Worker/FhirTransaction/LoggingFhirTransactionPipelineStep.cs

Reply:
Can you bring this up in the standup tomorrow. We can catch a global exception in DicomCastWorker and log them and throw them back.